src/OpenSage.Game/Logic/Object/GameObject.cs

```diff
@@ -157,6 +162,7 @@ public bool IsDamaged
         }
 
         public float Speed { get; set; }
+        public float Yaw { get; set; }
```

Comment:
Required for FrontWheelSteering

Reply:
I'm going to move `Speed` to `PhysicsBehavior` soon (and make it a `Vector3` velocity) so I'll move this too. But it's fine here for now.